app/src/main/java/edu/usf/sas/pal/muser/firebase/models/SongActionProvider.java

```diff
@@ -0,0 +1,16 @@
+package edu.usf.sas.pal.muser.firebase.models;
+
+/**
+ *  The provider class that stores different types of action that can be performed by the user
+ *  on a sound track.
+ */
+public interface SongActionProvider {
+    @interface ActionType{
```

Comment:
Same here - let's remove `SongActionProvider` wrapper and have a `EventType` interface

Reply:
Actually, this should be an `enum`, right?

Reply:
Initially, I referred to the `ArtworkProvider` class in the Shuttle code to implement this. I thought this reduced the number of lines of code and maintained a level of code similarity compared to using an enumerator. Please feel free to let me know your thoughts on using this. 

https://github.com/CUTR-at-USF/MUSER/blob/usf/app/src/main/java/com/simplecity/amp_library/model/ArtworkProvider.java#L12

Reply:
To make the Firebase data easy to read and not dependent on the Android code I'd actually prefer these get written as Strings to Firebase. So let's just make them enums to make that easier.